### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by garanews**
fix typo

### CODE REVIEW

File: docs/graphql/core/queries/query-filters.rst [hunk 1]
```diff
@@ -57,10 +57,10 @@ argument to filter on equality.
 
 You can see the complete specification of the ``where`` argument in the :ref:`API reference <WhereExp>`.
 
-Comparision operators
+Somparison operators
```
Comment (ShraddhaAg) [inline docs/graphql/core/queries/query-filters.rst:60]:
```suggestion
Comparison operators
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -329,7 +329,7 @@ using this flag is insecure since verification is not carried out.
 
 - console: update graphiql explorer to support operation transform (#4567)
 - console: make GraphiQL Explorer taking the whole viewport (#4553)
-- console: fix table columns type comparision during column edit (close #4125) (#4393)
+- console: fix table columns type comparison during column edit (close #4125) (#4393)
```
Comment (ShraddhaAg) [inline CHANGELOG.md:332]: This is quite an outdated entry. Could you please revert this change?

**Review by ShraddhaAg — CHANGES_REQUESTED** [review]
Thank you for the fixes @garanews! Left a few comments but overall this looks good 🥳

File: CHANGELOG.md (hunk 2 above)
Comment (garanews) [reply CHANGELOG.md:332]: revert leaving wrong?

Comment (ShraddhaAg) [reply CHANGELOG.md:332]: CHANGELOG shouldn't be edited for anything else other than adding an entry for the current PR if required. Please remove the changes you have made here, that'll be great :)

Comment (garanews) [reply CHANGELOG.md]: ok I did it, for your information on other repo I was able to modify changelog, and people suggested to add the new pr under improvement of current version. but you're the owner of your repo, so I will follow your suggestions :)

**Review by ShraddhaAg — APPROVED** [review]

**Review by marionschleifer — APPROVED** [review]

**Review by rikinsk — APPROVED** [review]

**Comment by mehtavishwa30** [comment]
Hello @garanews!

Thanks so much for your contribution to Hasura OSS during the Hacktoberfest 2020! We'd love to send you some Hasura swag as a token of appreciation. If you can share your email address, we will send you an email with the swag form shortly. ✨

**Comment by garanews** [comment]
@mehtavishwa30 sure:

puntogtg at tiscali dot it
